src/gitlab-api/index.ts

```diff
@@ -75,6 +75,19 @@ export async function getMergeRequestFiles(
         const parser = new XMLParser({
             ignoreAttributes: false,
             attributeNamePrefix: '',
+            tagValueProcessor: (
+                tagname,
+                tagValue,
+                jPath,
+                hasAttributes,
+                isLeadNode
+            ) => {
+                if (!isLeadNode || typeof tagValue !== 'string') return null
+                return tagValue
+                    .split('\n')
+                    .map((line) => line.trim())
+                    .join('\n')
+            },
```

Comment:
Minor nitpick: Can this be moved into a function (e.g.: into `src/lib/xml-protocol-transfomers.ts`) and imported from there in both of these instances?

Reply:
Is it ok to just create the parser there and `export const` it? I'm not entirely sure about the semantics of module exports, I assume it's not lazy and will create it as soon as the module is evaluated potentially when it's unnecessary, at the same time the place it's actually in at runtime currently `src/gitlab-api/index.ts > getMergeRequestFiles` creates a new parser per file, so it's probably not that much of a concern and will already be an improvement.

Reply:
Sure, that would work too, although I'm not entirely sure if the parser is "stateless" and can be reused for multiple `.parse()` invocations but I guess we'll find out if it's not :)